File: Modules/YourModules/work.py
import subprocess
import time

constTime = 1

subprocess.Popen(["flatpak", "run", "md.obsidian.Obsidian"])
time.sleep(constTime)
subprocess.Popen(["telegram-desktop"])
time.sleep(constTime)
subprocess.Popen(["firefox"])
time.sleep(constTime)
subprocess.Popen(["code"])
time.sleep(constTime)
subprocess.Popen(["AmneziaVPN"])

# В данном случае хорошо бы использовать import time, в противном случае приложения могут не успевать запускаться
# Чем слабее компьютер, тем больше времени надо выставить значение в "constTime" (в идеале)

File: Modules/YourModules/work.go
package main

import (
    "os/exec"
)

func main() {
    
    //добавьте или измените список запускаемых приложений
    programs := []string{"firefox", "code", "AmneziaVPN", "telegram-desktop", "md.obsidian.Obsidian"} 

    for _, program := range programs {

        cmd := exec.Command(program)
        cmd.Start()

    }

}
